Makefile

```diff
@@ -132,6 +133,15 @@ cypress-primitives: build-statics
 
 ## Cypress stuff used in local dev
 
+run-app-stable:
```

Comment:
Highlight #2 When testing with cypress I found it good to have a server that doesn't keep restarting in response to code changes or the test runner never really gets off the ground.

TBH, the real solution is to tweak nodemon to ignore any changes to cypress test files or artefacts generated by cypress. Will do that before merging